### ISSUE TITLE
no distinguishement for f-strings

### ISSUE THREAD

**Issue by nebulousdog** [atom/language-python#291]
https://www.python.org/dev/peps/pep-0498/#how-to-denote-f-strings

![image](https://user-images.githubusercontent.com/2218331/51796500-532a9500-21a9-11e9-88e7-0fffd670d54c.png)

no syntax highlighting for f-strings.

thx!

**Comment by nicktimko** [comment]
@nebulousdog can you show what a r(aw)-string appears as for you as well? I think the issue might be your theme, as I get something like the below.

![image](https://user-images.githubusercontent.com/5297556/54853380-c2aa9580-4cbd-11e9-81d1-4db97ad8cfc3.png)

Running the valid ones it turns into: `['plain', 'f-string = 108', 'raw\\string', 'compat unicode']`

What should it look like?

---

I'm on VScode:

Version: 1.32.3
Commit: a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4
Date: 2019-03-14T23:38:49.842Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.15.0-46-generic (Ubuntu 18.04)

With Python extension:

Name: Python
Id: ms-python.python
Description: Linting, Debugging (multi-threaded, remote), Intellisense, code formatting, refactoring, unit tests, snippets, and more.
Version: 2019.2.5558
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-python.python

**Comment by nebulousdog** [comment]
Hi! sorry for the late reply.

Here are some various screenshots in Atom of the included themes, and my `apm` output.

```bash
apm --version
apm  2.1.3
npm  6.2.0
node 8.9.3 x64
atom 1.36.0-beta1
python 2.7.15+
git 2.19.1
```

Atom Dark

![image](https://user-images.githubusercontent.com/2218331/55770605-c5d1af80-5a39-11e9-84d4-6a17f0ebf983.png)

Base16 Tomorrow Dark

![image](https://user-images.githubusercontent.com/2218331/55770705-29f47380-5a3a-11e9-90f0-39b0cf2217a1.png)

One Dark

![image](https://user-images.githubusercontent.com/2218331/55770724-3c6ead00-5a3a-11e9-8de3-4a9c9749e2e1.png)

Solarized Dark

![image](https://user-images.githubusercontent.com/2218331/55770741-4bedf600-5a3a-11e9-9f99-bc32d1fbf7cd.png)

**Comment by nebulousdog** [comment]
here are the class names found via the inspector:

![image](https://user-images.githubusercontent.com/2218331/55771094-7db38c80-5a3b-11e9-9ffe-f8389e5a5c7d.png)

```html
<div class="line cursor-line" data-screen-row="29">
  <span class="">
    <span class="syntax--source syntax--python">
      <span class="leading-whitespace indent-guide">    </span><span class="syntax--string syntax--quoted">f"another test<span class="syntax--meta syntax--embedded"><span class="syntax--punctuation syntax--section syntax--embedded">{</span>bbq<span class="syntax--punctuation syntax--section syntax--embedded">}</span></span>"</span>,
    </span>
  </span>
</div>
```

**Comment by nebulousdog** [comment]
oh! This may be tree-sitter related? I remembered I had turned off the tree-sitter options due to the conversation in this `atom/language-javascript` issue https://github.com/atom/language-javascript/issues/600#issuecomment-436448053. I swapped the option to "use" tree sitter parsers, closed atom, and re-opened and got this different highlighting from the same Syntax Theme as in my original post. I'm going to close this issue, because I believe the default option in Atom is to have this turned on. Thanks.
